server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -454,6 +454,11 @@ partition_interval=604800
 memtable_num_in_each_storage_group=10
 
 
+# max size for tag and attribute of one time series
+# the unit is byte
+tag_attribute_total_size=700
```

Comment:
Should  we mention this config in document [chapter 3.4](https://iotdb.apache.org/zh/UserGuide/Master/3-Server/4-Config%20Manual.html)?

Reply:
added